src/coreclr/src/tools/ILVerification/ILVerification.csproj

```diff
@@ -5,6 +5,7 @@
     <CLSCompliant>false</CLSCompliant>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+    <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
```

Comment:
My understanding was that resx files in the current directory would be added automatically if this property isn't set. Is that wrong?

Reply:
Without this line, `ILVerification.Strings.resources` was included automatically and then the build complained that `ILVerification.Strings.resources` is included twice. The second one comes from `ILVerification.projitems`.

I am mirroring the setup this has for CompileItems for EmbeddedResourceItems. (Introduced by #37388 to fix build breaks in tests.)

Reply:
Got it, missed the specification in `projitems`